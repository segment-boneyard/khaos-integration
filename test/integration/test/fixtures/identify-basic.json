{
  "input": {
    "type": "identify",
    "timestamp": "2014",
    "userId": "user-id",
    "traits": {
      "firstName": "John",
      "lastName": "Doe",
      "email": "jd@example.com"
    },
    "context": {
      "ip": "10.0.0.1"
    }
  },
  "output": {
    "type": "identify",
    "timestamp": "2014-01-01T00:00:00.000Z",
    "userId": "user-id",
    "traits": {
      "firstName": "John",
      "lastName": "Doe",
      "email": "jd@example.com"
    },
    "context": {
      "ip": "10.0.0.1"
    }
  }
}
