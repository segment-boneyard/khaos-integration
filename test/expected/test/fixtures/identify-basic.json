{
  "input": {
    "type": "identify",
    "userId": "user-id",
    "traits": {
      "firstName": "John",
      "lastName": "Doe",
      "email": "jd@example.com"
    },
    "context": {
      "ip": "10.0.0.1"
    }
  },
  "output": {
    "type": "identify",
    "userId": "user-id",
    "traits": {
      "firstName": "John",
      "lastName": "Doe",
      "email": "jd@example.com"
    },
    "context": {
      "ip": "10.0.0.1"
    }
  }
}
